Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.cpp

```diff
@@ -44,7 +44,7 @@ void ControlGroup::AddDeadzoneSetting(SettingValue<double>* value, double maximu
               // i18n: The percent symbol.
               _trans("%"),
               // i18n: Refers to the dead-zone setting of gamepad inputs.
-              _trans("Input strength to ignore.")},
+              _trans("Input strength to ignore and remap.")},
```

Comment:
Not sure if "dead zone" is _also_ used to _remap_ anything. Care to elaborate?

Reply:
The deadzone remaps the values after the deadzone.
If deadzone is 0.1 and you pass in 0.5, the outcome won't be 0.5, it will be 0.55, so there is a remap on going (applied differently depending on the kind of deadzone), just saying "ignore" is not enough in my opinion, it's actually misleading.

Reply:
While the change is technically correct, I think it's going to be hard for users to understand what you mean. I think you at the very least should expand the i18n comment to explain what's meant by remap, so that the string doesn't get mistranslated.

Reply:
I can do that. If anyone has got a better word than remap, I'm happy to change it (maybe scale?). "Ignore" alone is technically wrong.

Reply:
I didn't realize this also happened here, thanks for clarifying!